tests/checkings.test.js

```diff
@@ -0,0 +1,96 @@
+import HomePage from "../pages/HomePage";
+import SignIn from "../pages/SignIn";
+import CheckingPage from "../pages/CheckingPage"
+ 
+
+describe('Checkings', () => {
+it('Should take you to Checkings form', async ()=> {
+    await SignIn.open('bank')
+    let username = 'jsmith@demo.io'
+    let password = 'Demo123!'
+    await SignIn.logIn(username, password)
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await expect(HomePage.checkingTitle).toHaveText('Create Checking')
+})
+it('Should take you to Checkings View', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessViewCheckingForm()
+    await expect(HomePage.checkingTitle).toHaveText('View Checking Accounts')
+})
+it('Should see the three pages going forwards and backwards', async ()=> {
+    await CheckingPage.changeCheckingFormPageNext()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 11 to 20 of 24 entries')
+    await CheckingPage.changeCheckingFormPageNext()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 21 to 24 of 24 entries')
+    await CheckingPage.changeCheckingFormPagePrevious()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 11 to 20 of 24 entries')
+    await CheckingPage.changeCheckingFormPagePrevious()
+    await expect(CheckingPage.checkingTableTextInfo).toHaveText('Showing 1 to 10 of 24 entries')
+})
+it('Should filter the table with the word "Income"', async ()=> {
+    await CheckingPage.sendText(CheckingPage.checkingSearchInput, 'Income')
+    await expect(CheckingPage.checkingTableTextInfo).toHaveTextContaining('(filtered from 24 total entries)')
+})
+it('Should reset the data', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await CheckingPage.sendText(CheckingPage.checkingNameInput, '1234')
+    await CheckingPage.checkingReset()
+    await expect(CheckingPage.checkingNameInput).toHaveText('')
+})
+it('Should create a Standard Individual Checking', async ()=> {
+    await HomePage.displayCheckingMenu() 
```

Comment:
Every test about creating an account shares the same overall structure, only changing the parameters sent for the inputs and (maybe a bit more subtle) the type of accounts radiobuttons options. 

See if you can somehow parameterize the selection of account type with a method (maybe something that takes the options for the account as parameters, such as `createAccount(type, ownership, name, amount)`, **type** being a string that could be standard or interest, and **ownership** being if its individual or joint), and then use a **data driven** approach to only define it once, and then run it with every combination of parameters possible.

Reply:
How do I transform the string into the element it has to click?

Reply:
Use an if statement or ternary operator to implement it (preferably ternary operator for readability/shortness). Something like:
```javascript
string == "individual" ? await this.selectIndividualAccount() : await this.selectJointAccount()
```

BTW, I don't remember the exact method names you used for those methods, but this is the idea. If string is "individual", it executes first part of the expression, else the other part: Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator